hypothesis_bio/hypothesis_bio.py

```diff
@@ -41,6 +41,33 @@ def dna(
     return draw(text(alphabet=chars, min_size=min_size, max_size=max_size))
 
 
+@composite
+def rna(
+    draw,
+    allow_ambiguous=False,
+    allow_gaps=False,
+    allow_lowercase=False,
+    min_size=0,
+    max_size: Optional[int] = None,
+):
+    """Generates RNA sequences.
+
+    Arguments:
+    - `allow_ambiguous`: Whether ambiguous bases are permitted.
+    - `allow_gaps`: Whether a `-` may be in the RNA sequence.
+    - `allow_lowercase`: Whether lowercase characters should be used.
+    - `min_size`: The shortest RNA sequence to generate
+    - `max_size`: The longest RNA sequence to generate
+    """
+
+    chars = "AUCG" if not allow_ambiguous else "AUCGNTWSMKRYBDHV"
```

Comment:
Might be better if we use the ambiguous bases `dict` from `utilities`? And just replace the occurence of `T` with `U`?

Reply:
Hmm maybe but I don't think that would be efficient. The string representation is a more concise representation.

Reply:
Agreed a string representation is more efficient. I mean more for consistency across the project. If we decide to add/remove ambiguous bases if we do it in a central place then we only have to change in one place. I don't think the overhead of generating the string from a dictionary would be noticeable to a user?

Reply:
Hmm, it would if we are generating a lot of sequences and the string formation takes place every time.
Regardless, that is something that would need to be changed for DNA as well and it is a refactoring change we can worry about in a different PR. I think the refactoring is out of the scope for this PR.